src/replacements.js

```diff
@@ -103,18 +103,6 @@ function definitions() {
                 description: _("VLC is a media player that can play most media files")
             }
         },
-        {
-            regex: {
-                windows: /(lime|frost)wire.*.exe/i,
-                linux: /frostwire.*/i
-            },
-            appName: _("FrostTorrent"),
-            flatpakInfo: { remote: 'eos-apps', id: 'net.sourceforge.Frostwire' },
-            replacementInfo: {
-                appName: _("FrostTorrent"),
-                description: _("FrostTorrent is a file-sharing application like LimeWire")
```

Comment:
Please note in the commit message that we have dropped FrostTorrent from our repo.

Reply:
I made this change.